src/main/java/com/springweb/dv_spring_web_mongo/model/Project.java

```diff
@@ -28,4 +23,8 @@
     public ProjectDTO convertToDTO() {
         return new ProjectDTO(this.id, this.projectName);
     }
+
+    public ProjectCreateOrUpdateDTO convertToDTOSend() {
```

Comment:
The method is used only from tests. It doesn't make sense to keep it here. Why you cannot just create new instance in the test?